example/package-a/src/PackageA/Other/LoopBoot.hs

```diff
@@ -0,0 +1,8 @@
+module PackageA.Other.LoopBoot where
+
+import LoopNonBoot (LoopNonBoot(..))
```

Comment:
Maybe this should be `import PackageA.Other.LoopNonBoot (LoopNonBoot(..))` ?

Reply:
Yup, thanks